.github/workflows/pr.yaml

```diff
@@ -0,0 +1,39 @@
+on: pull_request
+name: PR Review
+jobs:
+  test-composer-files:
+    name: Validate composer
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Setup PHP, with composer and extensions
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.2
+          extensions: ctype, dom, iconv, json, zip, gd, soap
+          coverage: none
+          tools: composer:v2
+      - name: Validate composer files
+        run: |
+          composer validate composer.json
+
+  phpcs:
+    name: PHP - Check Coding Standards
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Setup PHP, with composer and extensions
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.2
+          extensions: ctype, dom, iconv, json, zip, gd, soap
+          coverage: none
+          tools: composer:v2
+      - name: Install Dependencies
```

Comment:
You should [Cache Composer Dependencies](https://github.com/shivammathur/setup-php#cache-composer-dependencies)

(This is a `@TODO` for almost all of our composer actions)

Reply:
👍  you really should look into the composer cache to speed up everything

Reply:
I'm not sure how to do this (correctly) and how it actually works.
According to https://github.com/shivammathur/setup-php#cache-extensions is caches dependencies between runs.
But does it also cache between jobs in a run or do we need to set up caching on both jobs (`test-composer-files` and `phpcs`)?
Are jobs always run in the order specified in `pr.yaml`? Or are they run in parallel?